tools/main.c

```diff
@@ -60,7 +60,7 @@ main (int   argc,
     case 1:
         execute_man (argv[0], envp);
         LOG_ERR ("failed to load manpage, check your environment / PATH\n");
-        /* fallthrough */
+        exit (1);
```

Comment:
There's a variety of ways to markup legitimate fallthrough cases (https://developers.redhat.com/blog/2017/03/10/wimplicit-fallthrough-in-gcc-7) and the existing /* fallthrough */ works, but only when passing additional compile parameters. The toolchain where I ran into this problem enables the warning without using -Wimplicit-fallthrough=2, so the comment was not recognized.

Just calling plain exit() here would be simpler than expecting people compiling tpm2.0-tools to pick a suitable level for -Wimplicit-fallthrough, but if you prefer the current code, then I can also check whether the default warning level can be relaxed a bit.